library/xwiki

```diff
@@ -3,22 +3,22 @@
 Maintainers: XWiki Core Dev Team <committers@xwiki.org> (@xwiki)
 GitRepo: https://github.com/xwiki-contrib/docker-xwiki.git
 
-Tags: 9, 9.11, 9.11.8, 9-mysql-tomcat, mysql-tomcat, lts-mysql-tomcat, lts-mysql, lts
+Tags: 10, 10.11, 10.11.2, 10-mysql-tomcat, mysql-tomcat, lts-mysql-tomcat, lts-mysql, lts
 Architectures: amd64
-GitCommit: 1617db38480798d515bcef595397e43518150dde
-Directory: 9/mysql-tomcat
+GitCommit: 17951cc427bde059e25eca752d31364e20db3ab9
+Directory: 10/mysql-tomcat
 
-Tags: 9-postgres-tomcat, 9.11-postgres-tomcat, 9.11.8-postgres-tomcat, postgres-tomcat, lts-postgres-tomcat, lts-postgres
+Tags: 10-postgres-tomcat, 10.11-postgres-tomcat, 10.11.2-postgres-tomcat, postgres-tomcat, lts-postgres-tomcat, lts-postgres
 Architectures: amd64, arm64v8
-GitCommit: 1617db38480798d515bcef595397e43518150dde
-Directory: 9/postgres-tomcat
+GitCommit: 17951cc427bde059e25eca752d31364e20db3ab9
+Directory: 10/postgres-tomcat
 
-Tags: 10, 10.11, 10-mysql-tomcat, 10.11-mysql-tomcat, stable-mysql-tomcat, stable-mysql, stable, latest
+Tags: 11, 11.0, 11.0.1, 11-mysql-tomcat, 11.0-mysql-tomcat, 11.0.2-mysql-tomcat, stable-mysql-tomcat, stable-mysql, stable, latest
```

Comment:
`11.0.1` vs `11.0.2-mysql-tomcat` :wink:

You are welcome to learn from any of our `generate-stackbrew-library.sh` scripts to help you generate this library file, like [cassandra](https://github.com/docker-library/cassandra/blob/197a99a985c50eefcf71ad6189764e7e81934596/generate-stackbrew-library.sh). I could make an initial pass if you like.

Reply:
Thanks for the suggestion, we'll look into it.